cpp/src/arrow/python/numpy_to_arrow.cc

```diff
@@ -1113,6 +1136,8 @@ Status NumPyConverter::ConvertObjects() {
     switch (type_->id()) {
       case Type::STRING:
         return ConvertObjectStrings();
+      case Type::BINARY:
+	return ConvertObjectBytes();
```

Comment:
This line looks misindented. Does your editor insert tabs instead of spaces?

Reply:
oops, let me fix that.